src/components/Sum.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = (props) => {
+  const { a, b } = props;
+
+  return (
+    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```

Comment:
I think you don't need to interpolate inside a `<p>` tag.

Reply:
agree, but one linter rule must be disabled. it's best to do that in eslint config file